portfolio/src/main/java/com/google/sps/data/Message.java

```diff
@@ -0,0 +1,40 @@
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.data;
+
+import java.util.Date;
+import java.text.SimpleDateFormat;
+
+/**
+ * <p>Note: The private variables in this class are converted into JSON.
```

Comment:
I don't fully understand this comment. When are they converted? How is this class supposed to be used?